cygrid/cygrid.pyx

```diff
@@ -900,7 +917,7 @@ cdef class SlGrid(Cygrid):
     ----------
     sl_lons, sl_lats : numpy.ndarray (float64), 1D
         Coordinates of sight lines to grid onto.
-    naxes3 : int
+    naxes3 : int (Default: 1)
```

Comment:
Could you be more verbose about the default being extracted from the header, and only set to 1 if there is no such entry?